node.go

```diff
@@ -54,10 +53,13 @@ func NewNode(name string, style code.StyleCode, ip net.IP, log Logger) *Node {
 		shutdown: true,
 		log:      log.With(Fields{"type": "Node"}),
 	}
-	if len(ip) > 0 {
-		n.Config.IP = ip
+
+	if len(ip) < 1 {
```

Comment:
Should we remove this check completely and add a validation + error on line 93, first line of `Start` method? You hardly want to start a node with a random IP, do you? As the IP is a required parameter we should just use this and leave the logic to the library user, what do you think?

Reply:
I would like to add static address generation following the scheme as depicted op p9 of the artnet specification (based on mac address)

For controllers this might not be common, but for other purposes it is.

Reply:
Ah, sounds sane. Fine for me :)